tgui/packages/tgui/interfaces/Wizard_Spellbook.js

```diff
@@ -105,32 +105,54 @@ const Spell = (props, context) => {
   const { spell, category } = props;
 
   const [purchased, setPurchased] = useSharedState(context, spell+"p", false);
-  let spell_contents = []; // Non-associated list of: desc, cost, cooldown, vr_allowed
+  let spell_contents = []; // Non-associated list of: desc, cost, cooldown, vr_allowed, spell_img
   for (let spell_data in spellbook_contents[category][spell]) {
     spell_contents.push(spellbook_contents[category][spell][spell_data]);
   }
 
   return (
     <Stack.Item>
-      <Section>
+      <Section mt={0.5} mb={-1.5}>
         {(vr === 1) && (spell_contents[3] === 0) && (
-          <Dimmer>
+          <Dimmer mb={3} mt={-2}>
             <Box fontSize={1.5} backgroundColor={"#384e68"} p={2}>
               Spell unavailable in VR
             </Box>
           </Dimmer>
         )}
         <Section
-          title={spell}
-          buttons={
-            <Flex>
-              <Button
-                backgroundColor={"green"}
-                disabled={spell_slots < spell_contents[1] || purchased}
-                onClick={() => { setPurchased(true); act("buyspell", { spell: spell }); }}
-              >
-                {If_Purchased_Text(purchased, spell_contents[1], spell_slots)}
-              </Button>
+          title={
+            <Flex justify={"space-between"} align={"end"}>
```

Comment:
You could probably structure this as a single `<Flex>` making use of `grow`. Something like:
```
<Flex align="end">
  {!!spell_contents[4] && (
    <Flex.Item>
      <Image ... />
    </Flex.Item>
  )
  <Flex.Item grow fontSize={1.25}>
    {spell}
   /Flex.Item>
   Flex.Item>
    <Button ... />
   /Flex.Item>
</Flex>
```

Reply:
Your right, didnt know the `grow` property could be used like that.